spec/lib/api_spec.rb

```diff
@@ -0,0 +1,15 @@
+describe Api do
+  context "model_to_collection" do
+    it "support classes" do
+      expect(Api.model_to_collection(Vm)).to eq(:vms)
+    end
+
+    it "support class names" do
```

Comment:
@abellotti I think, unless there is a clear use case for both, it might be a case of YAGNI. I think a method that supports both tends to give me anxiety over having to remember which one to use, and gives no feedback over whether I'm right or wrong